!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  Bf: 10.8529
  Br: 10.1507
  Cf: -1.1705
  Cr: -1.1705
  Df: 2785.4
  Dr: 3152.3
  Vy_max: 3.0
  YawRate_max: 50.0
  dMtv: 100.0
  dRd: 1.0
  diff_delta: 100.0
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      Bf: 10.8529
      Br: 10.1507
      Cf: -1.1705
      Cr: -1.1705
      Df: 2785.4
      Dr: 3152.3
      Vy_max: 3.0
      YawRate_max: 50.0
      dMtv: 100.0
      dRd: 1.0
      diff_delta: 100.0
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      latency: 5
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      q_mu: 0.1
      q_n: 5.0
      q_s: 10.0
      q_slip: 2.0
      state: true
      type: ''
    state: []
  latency: 5
  q_mu: 0.1
  q_n: 5.0
  q_s: 10.0
  q_slip: 2.0
state: []
